sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +582,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+def test_10375():
+    import numpy as np
+    m = Min(x, y, z)
+    l = lambdify([x,y,z], m)
```

Comment:
@moorepants  ,yeah we can translate like that,but how to change parameters to  list element during translation?If we add some checks inside lambdify.py to make it a  list, there is possibility that user want to use other 3 parameter for `numpy.min` means `axis,out,keepdims` if user choosed `module=numpy` ,then how to handle that ?